obspy/io/gcf/__init__.py

```diff
@@ -0,0 +1,30 @@
+"""
+obspy.io.gcf - Guralpe Compressed Format read support for ObsPy
+==========================================================
```

Comment:
What do you mean?

On Thu, Jun 9, 2016 at 2:48 PM, Elliott Sales de Andrade <
notifications@github.com> wrote:

> In obspy/io/gcf/__init__.py
> <https://github.com/obspy/obspy/pull/1449#discussion_r66528115>:
>
> > @@ -0,0 +1,30 @@
> > +"""
> > +obspy.io.gcf - Guralpe Compressed Format read support for ObsPy
> > +==========================================================
>
> This heading is not long enough.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/obspy/obspy/pull/1449/files/974a93d3f7083dd0f186664e2bbff4b7dbe9c22e#r66528115>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKICcoD_nPlms6HXucqjvDMhhdKo2eMEks5qKIoigaJpZM4IyR6B>
> .
>



-- 
Ran Novitsky Nof
Post-Doc Fellow
Berkeley Seismological  Laboratory
University of California Berkeley

Rm 209, McCone Hall
Mob: 1-805-418-0581
Web: http://ran.rnof.info